Turn/src/Store.cpp

```diff
@@ -3,13 +3,101 @@
 
 #include "..\include\Common.h"
 #include "..\include\Store.h"
+#include "..\include\ItemTypes.h"
 
 
 using namespace std;
 using namespace Common;
 
-void Store::StoreFront(Player* _Player){
-    ClearScreen();
-    cout << "Work in progress!" << endl;
-    system("PAUSE");
+const unsigned short arrowsCost = 200;
+const unsigned short bombCost = 100;
+const unsigned short potionCost = 150;
+const unsigned short whetstoneCost = 20;
+
+//Messages
+string lowCoinsMessage = " Insufficient Coins !";
+string invalidChoice = " Invalid Choice Entered !";
+
+void Store::StoreFront(Player* _Player) {
+
+	unsigned short choice = 0;
+	unsigned short coins;
+
+	do
+	{
+		ClearScreen();
+		coins = _Player->GetCoins();
+		cout << endl << " Jeremy's Tools Store\tPlayer: ";
```

Comment:
I suggest that `Player::DisplayInventory` is called here instead of just printing the player's coins. This way they know what it is in their inventory when they're buying something.